src/main/java/com/jcabi/s3/Bucket.java

```diff
@@ -108,7 +108,26 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
         String pfx) throws IOException;
 
     /**
-     * Prefixed.
```

Comment:
You removed too much :)
Please leave *Creates bucket with specified origin bucket and prefix.* and a single empty comment line after it.